package.json

```diff
@@ -50,15 +51,18 @@
     "cross-env": "^5.0.5",
     "css-loader": "^0.28.7",
     "eslint": "^4.18.2",
+    "eslint-config-prettier": "^2.9.0",
     "eslint-config-standard": "^11.0.0",
     "eslint-plugin-import": "^2.9.0",
     "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-promise": "^3.7.0",
     "eslint-plugin-standard": "^3.0.1",
     "eslint-plugin-vue": "^4.3.0",
     "file-loader": "^1.1.4",
     "jest": "^22.4.3",
     "node-sass": "^4.5.3",
+    "prettier": "1.11.1",
```

Comment:
The docs [recommend](https://prettier.io/docs/en/install.html) pinning to a specific version.